packages/expo-diag/src/index.ts

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+import { Command } from 'commander';
+import path from 'path';
+
+import { actionAsync } from './diagnosticsAsync';
+
+const packageJson = () => require('../package.json');
+
+const command = new Command(packageJson().name)
```

Comment:
Maybe use [arg](https://www.npmjs.com/package/arg) instead of commander here since it's so simple and we want the package as small as possible.

Reply:
Looked at `arg`. Would you implement `--help` manually when using `arg`?

Reply:
yes and `--version`, `-v`, etc.